### DISCUSSION TITLE
Add title to fuzzy picker box

### DISCUSSION THREAD

**Discussion by ArchUsr64** [helix-editor/helix#10186] [Ideas]
The picker box in it's current implementation doesn't provide any context about what is being picked for.  
Below is a screenshot of the picker on global-search for the term `Viewport`, there is no way to find what search term the picker is referring to once you hit Enter on the search option.  

![image](https://github.com/helix-editor/helix/assets/83179501/f59aadc1-c6cc-490e-88aa-a7f738098039)

The above issue persists for other picker related things like the references picker etc. A helpful title such as "Global search: Viewport" or "LSP References: SomeStruct" would be nice.

**Comment by nik-rev** [comment]
I have implemented this here: https://github.com/helix-editor/helix/pull/12520
